cmd/k8s-operator/egress-services.go

```diff
@@ -54,9 +54,9 @@ const (
 	// container. In practice this will be ports in range [3000 - 4000). The
 	// high range should make it easier to distinguish container ports from
 	// the tailnet target ports for debugging purposes (i.e when reading
-	// netfilter rules). The limit of 10000 is somewhat arbitrary, the
+	// netfilter rules). The limit of 1000 is somewhat arbitrary, the
 	// assumption is that this would not be hit in practice.
-	maxPorts = 10000
+	maxPorts = 1000
```

Comment:
Could we also fix the maxPorts const comment?

Reply:
Oops, thanks for the catch